vmware/container-service-extension
# [VCDA-2472 & VCDA-2545] Use version key in template yaml instead of depending on naming

To help us process your pull request efficiently, please include: 

* Template .yaml should contain a version key which will help identifying the template script structure
* Remove hard coding of script directory path. Use script_directory key in template.yaml

Testing done:
* Install CSE 3.1 using cookbook with version 1.0 -> should fail with proper error message
* Install CSE 3.1 using cookbook with version 2.0 and install a template

@rocknes @sahithi @sakthisunda @arunmk


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/container-service-extension/1052)
